### PR TITLE
transport: bump conn window size to 16 times RPC window size

### PR DESCRIPTION
**Pull request by asubiotto**

### CODE REVIEW

**Comment by asubiotto** [comment]
@andreimatei @petermattis

**Review by petermattis — APPROVED** [review]
LGTM

File: test/end2end_test.go [hunk 1]
```diff
@@ -1268,6 +1268,9 @@ func testUnknownHandler(t *testing.T, e env, unknownHandler grpc.StreamHandler)
 	}
 }
 
+// TestHealthCheckServingStatus fails with
+// "race detected during execution of test" when initialWindowSize and
```
Comment (andreimatei) [inline test/end2end_test.go]: I don't think we should put this comment in this fork... Will be un-needed noise for merges and such. Consider filing a bug upstream.

**Review by andreimatei — APPROVED** [review]
LGTM

**Comment by asubiotto** [comment]
Removed the comment. @andreimatei, I don't have write access so you will need to merge this for me.

**Comment by andreimatei** [comment]
Merged. Also added you and some people as collaborators on the repo.
Although if we make more changes, we should prolly move it under cockroach.

**Pull request merged**